INSTALL.md

```diff
@@ -141,65 +141,67 @@ packages necessary for building .debs, so be prepared to type your password for
+as the copy you'll make by following this guide is only suitable for testing.
 
-Since https://github.com/ReDetection/homebrew-qtox you can easily install qtox with homebrew 
-```bash
-brew install --HEAD ReDetection/qtox/qtox
-```
+Compiling qTox on OS X for development requires 3 tools, [Xcode](https://developer.apple.com/xcode/) and [Qt 5.4+](http://www.qt.io/qt5-4/), and [homebrew](http://brew.sh).
 
+###Required tools
 
-###OSX Full Install Guide
+First, let's install the dependencies
+* ```brew install git wget```
+* ``git clone https://github.com/tux3/qTox``
+* ```cd qTox```
 
-This guide is intended for people who wish to use an existing or new ProjectTox-Core installation separate to the bundled installation with qTox, if you do not wish to use a separate installation you can skip to the section titled 'Final Steps'.
+###Libraries required to compile
 
-Installation on OSX, isn't quite straight forward, here is a quick guide on how to install;
+Now we are in the qTox folder and need our library dependencies to actually build it.
 
-Note that qTox now requires OpenCV and OpenAL for video and audio.
+I've taken the time to prepare them automatically with our CI system so if you ever have issues redownload them.
 
-The first thing you need to do is install ProjectTox-Core with a/v support. Refer to the INSTALL guide in the PrjectTox-Core github repo.
+* ```wget https://jenkins.libtoxcore.so/job/qTox%20OS%20X/lastSuccessfulBuild/artifact/dep.zip```
+* ```unzip dep.zip```
 
-Next you need to download QtTools (http://qt-project.org/downloads), at the time of writing this is at version .3.0.
-Make sure you deselect all the unnecessary components from the 5.3 checkbox (iOS/Android libs) otherwise you will end up with a very large download.
+If you do not want to download our binaries, you must compile [opencv2](http://opencv.org), [toxcore](https://github.com/irungentoo/toxcore), [opus](https://www.opus-codec.org), [vpx](http://www.webmproject.org/tools/), [filteraudio](https://github.com/irungentoo/filteraudio), and our fork of [openal](https://github.com/irungentoo/openal-soft-tox) yourself with the prefix to the libs folder.
 
-Once that is installed you will most likely need to set the path for qmake. To do this, open up terminal and paste in the following;
+Please be aware that no one has ever successfully got this working outside of on our CI system, but we encourage you to try and provide instructions on how you did so if you do.
 
-```bash
-export PATH=/location/to/qmake/binary:$PATH
-```
+Please be aware that you shouldn't do this on your main Mac, as it's fairly hard to successfully do this without ruining a bunch of things in the process.
 
-For myself, the qmake binary was located in /Users/mouseym/Qt/5.3/clang_64/bin/.
+As everything from toxcore to filter audio to our special openal fork with echo cancelation is now installed.
```

Comment:
I'm not sure what this sentence means. Are you saying "every dependency is installed now" or "as every dependecy is installed, we can start compiling" ?